lib/src/intl_suggestors/intl_configs_migrator.dart

```diff
@@ -0,0 +1,35 @@
+import 'package:analyzer/dart/ast/ast.dart';
+import 'package:analyzer/dart/ast/visitor.dart';
+import 'package:codemod/codemod.dart';
+import 'package:file/file.dart';
+import 'package:over_react_codemod/src/intl_suggestors/utils.dart';
+
+const varsToCheck = ['displayName', 'name', 'title'];
+
+class ConfigsMigrator extends RecursiveAstVisitor with AstVisitingSuggestor {
```

Comment:
It looks like this suggestor will operate on any class with these variables, not just config classes.

Reply:
I do not understand why.   

Shouldn't this limit its scope?
```Dart
 @override
  bool shouldSkip(FileContext context) =>
      !context.path.contains('_config.dart');
```

Reply:
Oh I'm sorry, you're totally right. Never mind!